content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -1010,6 +1010,62 @@ is registered to handle that kind of data source. When a suitable populator is i
 responsibility of that populator controller to report Events that relate to volume creation and issues during
 the process.
 
+## Provision of PersistentVolumeClaims from VolumeSnasphot in other namespaces
+
+{{< feature-state for_k8s_version="v1.26" state="alpha" >}}
```

Comment:
I think it would be good to incorporate this section into the "Data source references" above.

Reply:
I see. Fixed https://github.com/kubernetes/website/pull/37802/commits/ae77a57a131507e2958d4ac11fe290c70392f66b